packages/components/src/components/dialog/dialog.stories.ts

```diff
@@ -57,271 +66,60 @@ export const Default = {
 };
 
 /**
- *  The dialog's headline as displayed in the header. If you need to display HTML, use the `headline` slot instead.
+ * Use the `open` attribute to show the dialog.
  */
 
-export const Headline = {
-  parameters: {
-    controls: { exclude: ['headline'] }
-  },
-  render: (args: any) => {
-    return html` <div style="height: 40vh;">
-      ${generateTemplate({
-        args,
-        constants: [
-          {
-            type: 'slot',
-            name: 'headline',
-            value: ''
-          },
-          {
-            type: 'attribute',
-            name: 'headline',
-            value: 'This headline is set via the headline attribute'
-          }
-        ]
-      })}
-    </div>`;
-  }
+export const Open = {
+  name: 'Open',
+  render: () => html`
+    <div class="h-[40vh]">
+      <sd-dialog id="dialog" open>
+        <h4 slot="headline" class="sd-headline sd-headline-3xl">Lorem ipsum</h4>
+        <p class="sd-paragraph">
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nibh justo ullamcorper odio tempor molestie phasellus
```

Comment:
Its only a small thing, but in design its:
`Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam.`